### PR TITLE
Onboarding language switch, more localizable strings, work on ru-ru translation

### PR DESCRIPTION
**Pull request by Yokayo**
* Added language switch to the onboarding window;
* Made more strings localizable in various places;
* Enhanced Russian translation: fixed some errors and clumsy wording, translated more text.

### CODE REVIEW

**Comment by Cohee1207** [comment]
I have to revise this. I don't like explicit calls to applyLocale. Some parts need to be migrated to template renderer instead.

**Comment by Yokayo** [comment]
Made a commit to move these dynamic HTML pieces to templates.
Well, tried to, at any rate. I have close to no experience with JS async so it might be cringy.

**Comment by Cohee1207** [comment]
How realistic would it be to extract prompt manager changes into a separate PR? These would be a bit too much for me to test in one PR.

**Comment by Yokayo** [comment]
I'll try not to mess up the branch.

**Comment by Yokayo** [comment]
I think I removed all changes that affected prompt manager. I'll bring them back in another PR.

**Comment by Cohee1207** [comment]
Thanks a lot.

File: public/script.js [hunk 1]
```diff
@@ -857,12 +857,12 @@ async function firstLoadInit() {
         throw new Error('Initialization failed');
     }
 
-    await getSystemMessages();
-    sendSystemMessage(system_message_types.WELCOME);
     await getClientVersion();
     await readSecretState();
-    await getSettings();
     initLocales();
+    await getSystemMessages();
```
Comment (Cohee1207) [inline public/script.js:864]: I'll have to see if this loading order actually works for all scenarios. The current order was learned the hard way.

Comment (Cohee1207) [reply public/script.js:864]: Okay, I see the issue now. Fixed it.

**Pull request merged**